music_assistant/providers/alexa/__init__.py

```diff
@@ -0,0 +1,314 @@
+            return
+
+        device_object = self.AlexaDevice()
+        await device_object.createobject(player_id, self.login)
+        api = AlexaAPI(device_object, self.login)
+        await api.set_volume(0)
+
+        player.volume_level = 0
+        self.mass.players.update(player_id)
+
+    async def cmd_seek(self, player_id: str, position: int) -> None:
+        """Handle SEEK command for given queue.
+
+        - player_id: player_id of the player to handle the command.
+        - position: position in seconds to seek to in the current playing item.
+        """
+
+    async def play_media(
+        self,
+        player_id: str,
+        media: PlayerMedia,
+    ) -> None:
+        """Handle PLAY MEDIA on given player.
+
+        This is called by the Players controller to start playing a mediaitem on the given player.
+        The provider's own implementation should work out how to handle this request.
+
+            - player_id: player_id of the player to handle the command.
+            - media: Details of the item that needs to be played on the player.
+        """
+        if not (player := self.mass.players.get(player_id)):
+            return
+
+        async with aiohttp.ClientSession() as session:
+            try:
+                async with session.post(
+                    "http://localhost:3000/ma/push-url",
+                    json={"streamUrl": media.uri},
+                    timeout=aiohttp.ClientTimeout(total=10),
+                ) as resp:
+                    await resp.text()
+            except Exception as exc:
+                _LOGGER.error("Failed to push URL to Alexa: %s", exc)
+                return
+
+        device_object = self.AlexaDevice()
+        await device_object.createobject(player_id, self.login)
+        api = AlexaAPI(device_object, self.login)
+        await api.run_custom("Ask music assistant to play audio")
+
+        player.current_media = media
+        player.elapsed_time = 0
+        player.elapsed_time_last_updated = time.time()
+        player.state = PlayerState.PLAYING
+        self.mass.players.update(player_id)
+
+    async def enqueue_next_media(self, player_id: str, media: PlayerMedia) -> None:
+        """
+        Handle enqueuing of the next (queue) item on the player.
+
+        Called when player reports it started buffering a queue item
+        and when the queue items updated.
+
+        A PlayerProvider implementation is in itself responsible for handling this
+        so that the queue items keep playing until its empty or the player stopped.
+
+        This will NOT be called if the end of the queue is reached (and repeat disabled).
+        This will NOT be called if the player is using flow mode to playback the queue.
+        """
+        device_object = self.AlexaDevice()
+        await device_object.createobject(player_id, self.login)
+        api = AlexaAPI(device_object, self.login)
+        await api.next()
```

Comment:
This seems incorrect. 
I think you simply need to use flow mode and set "enqueue" feature to false